js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "composio-core",
-  "version": "0.3.0",
+  "version": "0.3.1",
   "description": "",
```

Comment:
Consider adding a meaningful description for the package. An empty description field makes it harder for users to understand the package's purpose from the package.json metadata.